vividus-plugin-azure-cosmos-db/src/main/java/org/vividus/azure/cosmos/CosmosDbService.java

```diff
@@ -66,6 +68,16 @@ public CosmosContainer load(CosmosDbContainer cosmosDbContainer)
                         {
                             builder.key(key);
                         }
+
+                        if (GATEWAY.equals(account.getConnectionMode()))
+                        {
+                            builder.gatewayMode();
+                        }
+                        else
+                        {
+                            builder.directMode();
+                        }
+
```

Comment:
```suggestion

                        switch (account.getConnectionMode())
                        {
                            case GATEWAY -> builder.gatewayMode();
                            case DIRECT -> builder.directMode();
                        }

```

Reply:
Is it acceptable, if this will not cover case without specified `connection-mode` for CosmosDB account?